3 Domain layer/CandidatesEvaluator.Contract/Queries/Question/GetQuestion.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace CandidateEvaluator.Contract.Queries.Question
+{
+    public class GetQuestion : IQuery<Models.Question>
```

Comment:
Add namespace to using :)

Reply:
I cannot because this GetQuestion class is in Question namespace - I think we have to consider solving problems like that because in a few places we have classes, files, namespaces with the same names

Reply:
Ok, you have right :)